src/vs/workbench/api/common/extHostTerminalService.ts

```diff
@@ -892,15 +892,18 @@ class EnvironmentVariableCollection implements vscode.EnvironmentVariableCollect
 	}
 
 	replace(variable: string, value: string, scope?: vscode.EnvironmentVariableScope): void {
-		this._setIfDiffers(variable, { value, type: EnvironmentVariableMutatorType.Replace, scope });
+		// TODO: Implement options in API
+		this._setIfDiffers(variable, { value, type: EnvironmentVariableMutatorType.Replace, scope, options: {} });
 	}
 
 	append(variable: string, value: string, scope?: vscode.EnvironmentVariableScope): void {
-		this._setIfDiffers(variable, { value, type: EnvironmentVariableMutatorType.Append, scope });
+		// TODO: Implement options in API
+		this._setIfDiffers(variable, { value, type: EnvironmentVariableMutatorType.Append, scope, options: {} });
 	}
 
 	prepend(variable: string, value: string, scope?: vscode.EnvironmentVariableScope): void {
-		this._setIfDiffers(variable, { value, type: EnvironmentVariableMutatorType.Prepend, scope });
+		// TODO: Implement options in API
+		this._setIfDiffers(variable, { value, type: EnvironmentVariableMutatorType.Prepend, scope, options: {} });
```

Comment:
Currently blocked on a conflict with the old `scopes?` proposal from https://github.com/microsoft/vscode/issues/171173